README.rst

```diff
@@ -58,6 +60,9 @@ trying out materializations.
    # Run all catalog tests.
    $ flowctl test
 
+   # Start a local Flow instance and deploy the catalog to it
```

Comment:
Don't think I raised this directly with you: we've been using `flowctl deploy` colloquially, but that's not right. The operative term is to `apply`. As in `kubectl apply`, and `gazctl apply`.

"deploy" has meaning only in terms of standing up the Flow runtime, absent of any particular catalog.

Reply:
Good thought. I like `apply` a lot better